lib/src/pubspec.dart

```diff
@@ -630,6 +673,34 @@ class Pubspec {
         () => new VersionConstraint.parse(node.value));
   }
 
+  /// Parses [node] to a map from feature names to whether those features are
+  /// enabled.
+  Map<String, bool> _parseDependencyFeatures(YamlNode node) {
+    if (node?.value == null) return const {};
+    if (node is! YamlMap) _error('Features must be a map.', node.span);
+
+    return mapMap((node as YamlMap).nodes,
+        key: (nameNode, _) => _validateFeatureName(nameNode),
+        value: (_, valueNode) {
+          var value = valueNode.value;
+          if (value is bool && value) return value;
+          _error('Features must be set to true.', valueNode.span);
```

Comment:
Can you leave a comment explaining that this will be expanded later to allow other values? Right now, it looks silly to require `true`.